Widgets.go

```diff
@@ -113,6 +113,26 @@ func RadioButton(id string, active bool) bool {
 	return C.iggRadioButton(idArg, castBool(active)) != 0
 }
 
+// RadioButtonInt modifies integer v. Returns true if it is selected.
+//
+// The radio button will be set if v == button. Useful for groups of radio
+// buttons. In the example below, "radio b" will be selected.
+//
+//		v := 1
+//		imgui.RadioButtonInt("radio a", &v, 0)
+//		imgui.RadioButtonInt("radio b", &v, 1)
+//		imgui.RadioButtonInt("radio c", &v, 2)
+//
+func RadioButtonInt(id string, v *int, button int) bool {
+	idArg, idFin := wrapString(id)
+	defer idFin()
+	ok := C.iggRadioButtonInt(idArg, C.int(*v), C.int(button)) != 0
+	if ok {
```

Comment:
With the return value check, this implementation is now similar to that of the native one.
By this extend, I propose to implement it in Go as it is done in C++, by calling `iggRadioButton(char*, bool)` directly instead of checking the return value twice. This will make the changeset smaller as well.

Reply:
I don't understand what you're asking here. The return value check is so that the v value can be set.

Reply:
Yes, this is what I was going for. I forgot to refer to the actual ImGui implementation of that variant of `RadioButton()` to make it clear.
Thank you!